BMM.UI.Android/BMM.UI.Droid.csproj

```diff
@@ -3,7 +3,7 @@
   <PropertyGroup>
     <LangVersion>8</LangVersion>
     <AndroidSupportedAbis>armeabi-v7a;x86;arm64-v8a;x86_64</AndroidSupportedAbis>
-    <TargetFrameworkVersion>v11.0</TargetFrameworkVersion>
```

Comment:
In the last 30 days 22 devices have been using BMM. In January that number was still at 45.![Screen Shot 2022-06-15 at 2 55 37 PM](https://user-images.githubusercontent.com/649978/173904684-01afb8cf-bc20-4579-83ef-97c1fc1af6f7.png)
This chart show usage of Android 5 over the last 2 years.
Because of this I think it's reasonable to drop support of Android 5 now.